### PR TITLE
Update Kubermatic info

### PR DESCRIPTION
**Pull request by xmulligan**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/en/docs/setup/pick-right-solution.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ cluster.
 
 * [IBM Cloud Kubernetes Service](https://cloud.ibm.com/docs/containers?topic=containers-container_index#container_index) offers managed Kubernetes clusters with isolation choice, operational tools, integrated security insight into images and containers, and integration with Watson, IoT, and data.
 
-* [Kubermatic](https://www.loodse.com) provides managed Kubernetes clusters for various public clouds, including AWS and Digital Ocean, as well as on-premises with OpenStack integration.
+* [Kubermatic](https://www.loodse.com) runs Kuberntes in Kubernetes to provide managed Kubernetes clusters for any public cloud, including AWS and Digital Ocean, as well as on-premises with OpenStack integration.
```
Comment (bradtopol) [inline content/en/docs/setup/pick-right-solution.md]: Small typo: Kuberntes should be Kubernetes

**Comment by bradtopol** [comment]
small typo. see inline comments

**Comment by xmulligan** [comment]
Thanks @bradtopol just fixed it!

**Comment by bradtopol** [comment]
Great job!
/lgtm
/approve

**Pull request merged**